src/main/java/frc/robot/commands/VisionAssistedShooter.java

```diff
@@ -1,3 +1,4 @@
+package frc.robot.commands;
```

Comment:
Do you know why this line is needed when nothing else changed in the file?

Reply:
When I pulled from master it was somehow missing, so it caused an error.